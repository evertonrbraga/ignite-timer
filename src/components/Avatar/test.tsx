import { render, screen } from '@testing-library/react'

import { Avatar } from '.'

describe('<Avatar />', () => {
  it('should render the heading', () => {
    const { container } = render(<Avatar />)
    expect(screen.getByRole('heading', { name: /Avatar/i })).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})