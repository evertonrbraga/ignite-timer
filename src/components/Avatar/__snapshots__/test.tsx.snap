// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Avatar
  </h1>
</main>
`;
